tests/ExtraTests/X91-AmalgamatedCatch.cpp

```diff
@@ -14,7 +14,7 @@
 #include "catch_amalgamated.hpp"
 
 TEST_CASE("Just a dummy test") {
-    auto i = GENERATE(1, 2, 3);
+    [[maybe_unused]] auto i = GENERATE(1, 2, 3);
     SECTION("a") {
         REQUIRE(1 != 4);
```

Comment:
I think this might've been supposed to be `REQUIRE(i != 4)`.

Reply:
Done.